InformationScripting/src/parsing/QueryBuilder.cpp

```diff
@@ -81,12 +81,13 @@ QPair<QStringList, QList<QChar>> QueryBuilder::split(const QString& text, const
 	QPair<QStringList, QList<QChar>> result;
 	QString currentString;
 	QVector<int> openScopes(3, 0);
-	for (int i = 1; i < text.size() - 1; ++i)
+	for (int i = 2; i < text.size() - 2; ++i)
```

Comment:
It should be a class invariant that OPEN/CLOSE_SCOPE_SYMBOL have this length. Not sure how to implement this nicely.

Fix a constant and iterate at construction over the symbols and check their length?